lighthouse-core/gather/gather-runner.js

```diff
@@ -6,14 +6,17 @@
 'use strict';
 
 const log = require('lighthouse-logger');
-const manifestParser = require('../lib/manifest-parser.js');
-const stacksGatherer = require('../lib/stack-collector.js');
 const LHError = require('../lib/lh-error.js');
 const NetworkAnalyzer = require('../lib/dependency-graph/simulator/network-analyzer.js');
 const NetworkRecorder = require('../lib/network-recorder.js');
 const constants = require('../config/constants.js');
 const i18n = require('../lib/i18n/i18n.js');
 const URL = require('../lib/url-shim.js');
+const {getBenchmarkIndex} = require('./driver/environment.js');
```

Comment:
not using `getBrowserVersion` here:

```
const hostUserAgent = (await options.driver.getBrowserVersion()).userAgent;
```

Is that on purpose?

Reply:
eh yeah might as well remove this one too 👍

Reply:
nah I take it back the amount of test churn for this one change isn't worth it to combine, can be saved for eventual cleanup :)